Biotech/Patches/DamageDefs/Damages_Misc.xml

```diff
@@ -36,4 +36,15 @@
 		</value>
 	</Operation>
 
+	<!-- ========== ToxGas ========== -->
+
+	<Operation Class="PatchOperationAddModExtension">
+		<xpath>Defs/DamageDef[defName="ToxGas"]</xpath>
+		<value>
+			<li Class="CombatExtended.DamageDefExtensionCE">
```

Comment:
I honestly think this is probably something we'd prefer to add to the ammo item, rather than the DamageDef itself--that leaves us finer control over what does or doesn't spawn for mortar shells dropped to sieges. Thoughts?

Reply:
Yeah I don't really like how currently we can't control it per shell, was originally done this way to stay similar to the old/vanilla method of just checking damage but it's definitely gotten bloated/overcomplicated. What do you think about scrapping the damage/harmshealth checks and instead adding something like `SpawnAsSiegeAmmo` to the AmmoDef class? That way we can just blacklist shells we don't want since I think default behaviour should be to include them

Reply:
Yeah, I think an easy true/false we can just assign to each shell as an override is probably our easiest way to address edge cases. Leave the more straight-forward checks in the code to filter out shells that _obviously_ aren't suitable for siege, but we'll just use xml to address any outliers as necessary. Otherwise, we end up with a hideous, expensive string of checks that _still_ will find a way to let some strange shells spawn when they shouldn't.